lib/tests.js

```diff
@@ -228,6 +228,25 @@ tests.androidLongTest = async function (driver, caps) {
   }
 };
 
+tests.androidLoadTest = async function (driver, caps) {
+  let testSeconds = 600;
+  let args = await driver.elementById('textOptions');
+  let goBtn = await driver.elementById('go');
+
+  await args.clear();
+  await args.sendKeys("-M 500 -v 20" + " -s " + testSeconds);
+
+  for (let test_minutes = testSeconds / 60; test_minutes > 0; test_minutes--) {
+    await driver.sleep(30000);
```

Comment:
might be nice to have a comment to explain the sleeps, like `// here the app is doing its stressy thing so we wait`.

also, why is the sleep _before_ you click the button? doesn't that just add 30s of wait time up front?

Reply:
Good point, I've cleaned that up. I've also added a comment